docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/snow-create.md

```diff
@@ -0,0 +1,379 @@
+          }}:${{ imageFrom(vars.imageRepo).Tag }}
+        urgency: "3"
+      tableName: change_request
+      template:
+        templateId: ed89b72c83c172104517e470ceaad30a
+        type: standard
+    uses: snow-create
+  - as: snowquery
+    config:
+      credentials:
+        namespace: kargo-demo
+        secretName: snow-creds
+        type: api-token
+      query:
+        number: ${{ task.outputs.snowcreate.number }}
+      tableName: change_request
+    uses: snow-query-records
+  - as: snowupdate
+    config:
+      credentials:
+        secretName: snow-creds
+        type: api-token
+      parameters:
+        short_description: Update deployment for ${{ vars.imageRepo }}:${{
+          imageFrom(vars.imageRepo).Tag }} to resolved
+      tableName: change_request
+      template:
+        type: standard
+      ticketId: ${{ task.outputs.snowquery.sys_id }}
+    uses: snow-update
+  - as: snow-wait-for-condition
+    config:
+      condition: state=-2
+      credentials:
+        namespace: kargo-demo
+        secretName: snow-creds
+        type: api-token
+      tableName: change_request
+      ticketId: ${{ task.outputs.snowquery.sys_id }}
+    uses: snow-wait-for-condition
+  - as: snowdelete
+    config:
+      credentials:
+        secretName: snow-creds
+        type: api-token
+      tableName: change_request
+      template:
+        type: standard
+      ticketId: ${{ task.outputs.snowquery.sys_id }}
+    uses: snow-delete
+```
+
+### Table API Workflow
+
+```yaml
+  - as: snowcreate
+    config:
+      credentials:
+        secretName: snow-creds
+        type: api-token
+      parameters:
+        impact: "3"
+        short_description: Deploy to ${{ ctx.stage }} complete for ${{ vars.imageRepo }}:${{ imageFrom(vars.imageRepo).Tag }}
+        urgency: "3"
+      tableName: incident
+    uses: snow-create
+  - as: snowquery
+    config:
+      credentials:
+        namespace: kargo-demo
+        secretName: snow-creds
+        type: api-token
+      query:
+        number: ${{ task.outputs.snowcreate.number }}
+      tableName: incident
+    uses: snow-query-records
+  - config:
+      ticketId: ${{ task.outputs.snowquery.sys_id }}
+      credentials:
+        secretName: snow-creds
+        type: api-token
+      parameters:
+        short_description: Update deployment for ${{ vars.imageRepo }}:${{ imageFrom(vars.imageRepo).Tag }} to resolved
+      tableName: incident
+    uses: snow-update
+  - as: snow-wait-for-condition
+    config:
+      condition: state=6
+      credentials:
+        namespace: kargo-demo
+        secretName: snow-creds
+        type: api-token
+      tableName: incident
+      ticketId: ${{ task.outputs.snowquery.sys_id }}
+    uses: snow-wait-for-condition
+  - as: snowdelete
+    config:
+      credentials:
+        secretName: snow-creds
+        type: api-token
+      tableName: incident
+      ticketId: ${{ task.outputs.snowquery.sys_id }}
+    uses: snow-delete
+```
```

Comment:
don't think we need this, the change request example is enough it does almost the same